### PR TITLE
feat(api): add run result of type coverage

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-cds#20159.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20159.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

File: engine/worker/pkg/workerruntime/types.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ type V2UpdateResultResponse struct {
 
 type V2FilterRunResult struct {
 	Pattern              string
-	Type                 sdk.V2WorkflowRunResultType
+	Type                 []string
```
Comment (fsamin) [inline engine/worker/pkg/workerruntime/types.go:41]:
```suggestion
	Type                 []sdk.V2WorkflowRunResultType
```

**Review by fsamin — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20163.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by yesnault — APPROVED** [review]

**Review by fsamin — APPROVED** [review]

**Pull request merged**